### ISSUE TITLE
soft crossentropy loss 的加入

### ISSUE THREAD

**Issue by zhijiejia** [PaddlePaddle/PaddleSeg#1771]
请问，有考虑加入 soft crossentropy loss , 这个对用paddleseg参加比赛会有些用处？

paddle中的scatter 和 pytorch中的scatter 感觉功能不一样，如果需要 [H, W] -> [C, H, W]，用于生成soft label，在paddle中对应的函数是哪个啊？

**Comment by shiyutang** [comment]
We have not yet considered adding soft CELoss. But we very much welcome you contributing it to our repo.
It seems F.one_hot could be useful to you. Here is a [example](
https://github.com/PaddlePaddle/PaddleSeg/blob/4aa737e2648355994efe762edc1382f802b9816c/paddleseg/models/losses/rmi_loss.py#L103):
![image](https://user-images.githubusercontent.com/34859558/154232832-c7c98b40-e6f5-4d5a-a91e-6e48d4a65aa8.png)

**Comment by shiyutang** [comment]
If you have any further questions, you are very welcome to reopen it.
